source/retrieve-data.txt

```diff
@@ -0,0 +1,196 @@
+.. _java-rs-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. You can call the
+``find()`` method to retrieve documents that match a set of criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+.. _java-rs-retrieve-find:
+
+Find Documents
+--------------
+
+The {+driver-short+} includes one method for retrieving documents from a collection:
+``find()``.
+
+This method takes a **query filter** and returns one or more matching documents.
+A query filter is an object that specifies the documents you want to retrieve in
+your query. 
+
+.. TODO: uncomment when specify query is merged in
+.. To learn more about query filters, see :ref:`java-rs-specify-query`.
+
+.. _java-rs-retrieve-find-one:
+
+Find One Document
+~~~~~~~~~~~~~~~~~
+
+To find a single document in a collection, call the ``find()`` method and pass a query
+filter that specifies the criteria of the document you want to find. Then, chain
+the ``first()`` method to find a single document. If more than one document matches the query
```

Comment:
I: the opening clause already states the goal ("to find a single document..."), so i think you can leave it off of the end of the second sentence. it also sounds a little strange to say "chain the X method" without specifying a target. since the previous section just said to pass a query filter, maybe something like this would work better:

```suggestion
To find a single document in a collection, chain the ``first()`` method to your
``find()`` method call. and pass a query. If more than one document matches the query
```

Edit: after reading further, I see that removing the bit about the query filter breaks the parallel structure of find one and find multiple. one option is to put the Find Multiple section first, since it's technically simpler than Find One. then you could introduce Find One by saying something like, "Call find() as above, but also chain the first() method..."

Reply:
I'm not sure about putting find multiple first; keeping in line with other standardized pages, we start with "insert/retrieve/etc one" first and then multiple. i reworded it to be more in keeping with your suggestion though.